docs/third-party-tools/openbrush.md

```diff
@@ -33,9 +33,24 @@ code base ([detailed description](https://github.com/727-Ventures/openbrush-cont
 
 ### Wrapper around Traits
 
-OpenBrush simplifies cross-contract calls, a contract that implements a particular
-Trait is not needed to call it. A wrapper enables calling methods of that Trait
-from some contract in the network (do a cross contract call).
+Traditionally, if you want to do a cross-contract calling in ink!,
+you need to import the other contract as a dependency in the calling contract's project. 
+
+OpenBrush simplifies the process with the wrapper around traits. 
```

Comment:
Can you clarify here: 
* What is meant with "the wrapper"?
* Why does it need to be simplified in the first place?
* Is it just a simplification or does "The Wrapper" add guarantees?